lean/main/intro.lean

```diff
@@ -131,15 +131,15 @@ elab "#assertType " termStx:term " : " typeStx:term : command =>
 #assertType 5  : Nat -- success
 #assertType [] : Nat -- failure
 
-/-! We started by using `elab` to define a `command` syntax, which, when parsed
+/-! We started by using `elab` to define a `command` syntax, which when parsed
```

Comment:
This comma change too is a bit awkward to me -- the original sentence is kind of jaggedy (very technical term), so I get wanting to remove the comma, but to me if it removes this one I'd also remove the other one, i.e. "We started by using `elab` to define a `command` syntax, which when parsed by the compiler will trigger the incoming computation".

Reply:
Rewrote the sentence a bit. Let me know what you think.

Reply:
Thumbs up!